Resources/Prototypes/Entities/Objects/Consumable/Food/Containers/box.yml

```diff
@@ -274,6 +274,17 @@
       prob: 0.10
       orGroup: Pizza
     - id: KnifePlastic
+  
+- type: entity
+  name: pizza box
+  parent: FoodBoxPizzaFilled
+  id: FoodBoxPizzaCotton
+  suffix: Filled
```

Comment:
Change this so it's clearly shown as a cotton pizza in the spawn menu.

Reply:
This change has been pushed